dali/operators/reader/numpy_reader_op.cc

```diff
@@ -49,21 +49,21 @@ void NumpyReader::TransposeHelper(Tensor<CPUBackend>& output, const Tensor<CPUBa
 DALI_REGISTER_OPERATOR(NumpyReader, NumpyReader, CPU);
 
 DALI_SCHEMA(NumpyReader)
-  .DocStr("Read Numpy arrays from a directory")
+  .DocStr("Reads the Numpy arrays from a directory.")
   .NumInput(0)
   .NumOutput(1)  // (Arrays)
   .AddArg("file_root",
-      R"code(Path to a directory containing data files.
-`NumpyReader` supports flat directory structure. `file_root` directory should contain
-directories with numpy files in them.)code",
-      DALI_STRING)
+      R"code(Path to a directory containing the data files.
+
+:meth:`nvidia.dali.ops.NumpyReader` supports flat directory structure. The ``file_root`` directory
```

Comment:
I am not sure whether the full reference to this operator in its own documentation makes sense

Reply:
Removed